selfdrive/camerad/cameras/camera_webcam.cc

```diff
@@ -19,8 +19,8 @@
 #include "selfdrive/common/util.h"
 
 // id of the video capturing device
-const int ROAD_CAMERA_ID = util::getenv("ROADCAM_ID", 1);
-const int DRIVER_CAMERA_ID = util::getenv("DRIVERCAM_ID", 2);
+const int ROAD_CAMERA_ID = util::getenv("ROADCAM_ID", 4);
+const int DRIVER_CAMERA_ID = util::getenv("DRIVERCAM_ID", 0);
```

Comment:
Setting `ROADCAM_ID` and `DRIVERCAM_ID` can also reach the same result with these modification. Just for reference.